osvr/RenderKit/osvr_display_configuration.h

```diff
@@ -69,6 +70,14 @@ class OSVRDisplayConfiguration {
     void OSVR_RENDERMANAGER_EXPORT
     parse(const std::string& display_description);
 
+    /// Produces a duplicate object that leaves out/defaults the parameters related to display output transformation
+    /// (e.g. rotation, swap eyes).
+    ///
+    /// @returns a unique_ptr because that's what factory functions that return reference objects are recommended to do
+    /// (they're convertable to shared_ptr, but not vice versa)
+    std::unique_ptr<OSVRDisplayConfiguration>
+    OSVR_RENDERMANAGER_EXPORT duplicateWithoutOutputTransforms() const;
```

Comment:
This is called from inside the TS-DisplayServer's Singleton rendering class so that it can find the non-transformed data to have the client render with.  It then transforms it on the way to the display.